src/plugins/template/tests/functional/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ ov_add_test_target(
         DEPENDENCIES
             openvino_template_plugin
         LINK_LIBRARIES
-            openvino::funcSharedTests
```

Comment:
why?
All OpenVINO Dev targets in `plugin/template` must be with `openvino::` prefix 
See 
https://github.com/openvinotoolkit/openvino/blob/dbc911129a86dc9321ef5b2bb9feea43e81ddd41/cmake/extra_modules.cmake#L109-L130

Reply:
Yes. But @vurusovs reported that he has problems with this prefix in local builds:
`CMake Error at cmake/developer_package/add_target_helpers.cmake:109 (target_link_libraries):
  Target "ov_template_func_tests" links to:  openvino::funcSharedTests but the target was not found`

Other includes of funcSharedTests are used w/o openvino::prefix. So I decided to remove it from this place as well